src/views/CashlinkCreate.vue

```diff
@@ -584,26 +616,26 @@ export default class CashlinkCreate extends Vue {
     }
 
     /* these elements are too small to make a notable difference in the blurred background */
-    .create-cashlink .page-body >>> .arrow,
-    .create-cashlink .overlay ~ .page-header >>> a { /* back button and fee button */
+    .create-cashlink.blurred > .page-body >>> .arrow,
+    .create-cashlink.blurred > .page-header >>> a { /* back button and fee button */
```

Comment:
Added the correct rule for the `.options-button`. I choose to not blur it to smoothen the transition wherever possible and it does not show at all (at least when I look at it) in the final render of the overlay.

Reply:
As it sits within the `h1` that gets blurred, it gets blurred nonetheless.
Additionally, it changes the opacity as a nested transition which makes the transition even more expensive.
Either do not change the opacity and just let it get blurred or target the header text more specifically to avoid blurring the button.